src/shared/components/results/ResultsFacets.tsx

```diff
@@ -8,11 +15,44 @@ import TaxonomyFacet from './TaxonomyFacet';
 import { mainNamespaces } from '../../types/namespaces';
 
 import { UseDataAPIWithStaleState } from '../../hooks/useDataApiWithStale';
-import Response from '../../../uniprotkb/types/responseTypes';
+import Response, { FacetValue } from '../../../uniprotkb/types/responseTypes';
 
 import helper from '../../styles/helper.module.scss';
 import './styles/results-data.scss';
 
+const ICON_HEIGHT = '1em';
+
+const getDecoratedFacetLabel = (facetValue: FacetValue) => {
+  switch (facetValue.value) {
+    case 'true':
+      return (
+        <>
+          <SwissProtIcon height={ICON_HEIGHT} className="icon--reviewed" />
+          {facetValue.label}
+        </>
+      );
+    case 'false':
+      return (
+        <>
+          <TremblIcon height={ICON_HEIGHT} className="icon--unreviewed" />
+          {facetValue.label}
+        </>
+      );
+    case '1':
+      return (
+        <>
+          <ReferenceProteomeIcon
+            height={ICON_HEIGHT}
+            className="icon--reference-proteome"
+          />
+          {facetValue.label}
+        </>
+      );
+    default:
+      return facetValue.label;
+  }
+};
```

Comment:
Maybe investigate using the EntryTypeIcon with the label to get an Icon (or nothing) and inject it in front of the label

Reply:
We didn't think about the fact that `facetValue.label` is actually a `ReactNode`, not a string so `EntryTypeIcon` doesn't apply

Reply:
At the moment you receive it from the server it can only be a string.
I guess we switched the type to node so that we could use the same type for server-sent labels and client-side or customised label.
A type check of `typeof label === 'string'` should be enough to make TS happy

Reply:
Good idea, will try that.